dataprofiler/tests/profilers/profiler_options/test_categorical_options.py

```diff
@@ -1,6 +1,6 @@
 from dataprofiler.profilers.profiler_options import CategoricalOptions
-from dataprofiler.tests.profilers.profiler_options.test_base_inspector_options \
-     import TestBaseInspectorOptions
+from dataprofiler.tests.profilers.profiler_options.test_base_inspector_options import \
```

Comment:
since these are in tests, should we just be calling the tests relative as oppose to from the base of the dataprofiler. 

e.g. tests are relative, but repo itself is not? would fix these 80 char issues